app/views/ckey2bibframes/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1>Ckey to Bibframe2 Conversion</h1>
+
+<%= form_for(@ckey2bibframe) do |f| %>
+<% if @ckey2bibframe.errors.any? %>
+<div id="error_explanation">
+  <ul>
+    <% @ckey2bibframe.errors.full_messages.each do |message| %>
+    <li><%= message %></li>
+    <% end %>
+  </ul>
+</div>
+<% end %>
```

Comment:
Does the rspec test suite exercise this unhappy path for the "error explanation"?